lib/context.js

```diff
@@ -23,7 +23,7 @@ class Context {
       description.predicate = json
     } else {
       description.predicate = json['@id']
-      description.options.array = '@array' in json && json['@array']
+      description.options.array = '@container' in json && json['@container'] === '@set'
```

Comment:
Yes, `'@container': '@list'` would be a great addition to handle real RDF lists. I used `@set` here since the current array mechanism just deals with repeated values for a property.

Adding `@list` support dealing with those details requires more plumbing, so I suggest creating a specific issue for it. That is, the ordered aspect of arrays needs to reflect, and be reflected in, the underlying RDF list data (represented by the `rdf:first | rdf:rest` form).